setup.cfg

```diff
@@ -9,5 +9,4 @@ line_length = 88
 multi_line_output = 3
 use_parentheses = True
 lines_after_imports = 2
-known_third_party = PIL,bs4,classy_vision,cv2,detectron2,fairscale,faiss,fvcore,hydra,mock,nbconvert,nbformat,numpy,omegaconf,parameterized,pkg_resources,pycocotools,recommonmark,run_distributed_engines,scipy,setuptools,sklearn,sphinx_rtd_theme,tabulate,torch,torchvision,tqdm,utils
```

Comment:
is this change enforced by isort 5.7?

Reply:
I checked it out and it turns out it is not, I had just assumed (incorrectly) that since `seed-isort-config` was unnecessary that the option would be deprecated.